src/pages/ProductDesc/ProductDesc.js

```diff
@@ -1,8 +1,115 @@
 import React from "react";
 import "./ProductDesc.scss";
+import { useState, useEffect } from "react";
+import RecommendProducts from "./components/RecommendProducts/RecommendProducts";
+import NewProducts from "./components/NewProducts/NewProducts";
 
 const ProductDesc = () => {
-  return <div>ProductDesc입니다.</div>;
+  useEffect(() => {
+    fetch("/datas/RECOMMENDPRODUCTS_DATA.json", {
+      method: "GET",
+    })
+      .then(res => res.json())
+      .then(RECOMMENDPRODUCTS_DATA => {
+        setRecommendProductsData(RECOMMENDPRODUCTS_DATA);
+      });
+  }, []);
+
+  const [recommendProductsData, setRecommendProductsData] = useState([]);
```

Comment:
네이밍할 때 data는 빼주셔도 됩니다.
```suggestion
  const [recommendProducts, setRecommendProducts] = useState([]);
```